soco/core.py

```diff
@@ -1255,6 +1266,34 @@ def get_sonos_playlists(self, *args, **kwargs):
         args = tuple(['sonos_playlists'] + list(args))
         return self.get_music_library_information(*args, **kwargs)
 
+    def get_sonos_playlist_by_attr(self, attr_name, match):
+        """ Return the first Sonos Playlist DidlPlaylistContainer that
+        matches the attribute specified.
+
+        Args:
+            attr_name (str): DidlPlaylistContainer attribute to compare.
```

Comment:
I've modified the string to:
            attr_name (str): DidlPlaylistContainer attribute to compare.
The
                most useful being: 'title' and 'item_id'.


On Mon, Sep 7, 2015 at 10:46 AM, Lawrence Akka <notifications@github.com>
wrote:

> In soco/core.py
> <https://github.com/SoCo/SoCo/pull/352#discussion_r38870587>:
>
> > @@ -1255,6 +1266,34 @@ def get_sonos_playlists(self, *args, **kwargs):
> >          args = tuple(['sonos_playlists'] + list(args))
> >          return self.get_music_library_information(*args, **kwargs)
> >
> > +    def get_sonos_playlist_by_attr(self, attr_name, match):
> > +        """ Return the first Sonos Playlist DidlPlaylistContainer that
> > +        matches the attribute specified.
> > +
> > +        Args:
> > +            attr_name (str): DidlPlaylistContainer attribute to compare.
>
> "e.g. title, or ..." would be useful here
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/SoCo/SoCo/pull/352/files#r38870587>.
>



-- 
Best,

Jeff Hinrichs
dundeemt@gmail.com
402.218.1473